packages/cli/src/commands/hydrogen/setup.ts

```diff
@@ -0,0 +1,73 @@
+import Command from '@shopify/cli-kit/node/base-command';
+import {path} from '@shopify/cli-kit';
+import Flags from '@oclif/core/lib/flags.js';
+import {commonFlags} from '../../utils/flags.js';
+import {isGitClean} from '../../utils/git.js';
+import setupSeo from '../../setups/seo/seo.js';
+
+import {AbortError} from '@shopify/cli-kit/node/error';
+
+const SETUPS = ['seo', 'analytics', 'tailwind'];
+
+// @ts-ignore
+export default class Upgrade extends Command {
+  static description = 'Apply steps to setup various components.';
+  static flags = {
+    path: commonFlags.path,
+    force: commonFlags.force,
+    silent: Flags.boolean({
+      name: 'silent',
+      description: `Don't print any output.`,
+      env: 'SHOPIFY_HYDROGEN_FLAG_SILENT',
+    }),
+    dry: Flags.boolean({
```

Comment:
`--dry-run`/`-n` are the flags that I see most often for this behavior.

Reply:
Haha I agree. I was dancing around having to deal with camelcase 🗡️ 🗡️